frontend/src/app/pages/user-playlists/user-playlists.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component } from '@angular/core';
```

Comment:
idk if i like the idea of having two user-playlist and playlist but we can leave it as it is